packages/security/src/components/GroupDetail.vue

```diff
@@ -53,6 +53,51 @@
         </router-link>
       </div>
     </div>
+
+    <div class="row">
+      <div class="col">
+        <h3 class="mt-4">Permissions</h3>
+      </div>
+    </div>
+
+    <div class="jmbotron">
+      <div class="row groups-listing">
+        <div class="col">
+          <div class="form-check">
+            <input class="form-check-input" type="checkbox" id="AnonymousView" v-model="anonymousViewerPermission" :disabled="isSaving">
```

Comment:
Is de conventie niet om voor attributen lower-case + '-' te gebruiken? (zie ook andere radio buttons)